rpcs3/Emu/Cell/SPUThread.cpp

```diff
@@ -1463,6 +1490,20 @@ bool SPUThread::set_ch_value(u32 ch, u32 value)
 		ch_dec_value = value;
 		return true;
 	}
+	
+	case MFC_WrMSSyncReq:
```

Comment:
Reading through CBEA_v1.01_3Oct2006.pdf, this channel is supposed to stall if a second write happens to it before the first one is acknowledged. Also if there is no transfers pending, the multisource event should trigger instantly